posts/2020-09-25-signed-certificate-with-acme-200010.adoc

```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-title: "Install a signed certificate with ACME Certificate Authority with acmeCA-2.0"
+title: "Install a signed certificate with ACME Certificate Authority with acmeCA-2.0 and significant bug fixes on Open Liberty 20.0.0.10"
```

Comment:
I think also remove "with acmeCA-2.0" as that's just our name for the feature. The important bit is what the new version allows you to do with it.

Reply:
Done